README.md

```diff
@@ -12,13 +13,15 @@ for Android & Chrome OS ([beta](https://play.google.com/apps/testing/com.github.
 <a href="https://play.google.com/store/apps/details?id=com.github.shadowsocks.tv"><img src="https://play.google.com/intl/en_us/badges/images/generic/en-play-badge.png" height="48"></a>
 for Android TV ([beta](https://play.google.com/apps/testing/com.github.shadowsocks.tv))
 
+<img src='walkthrough.gif' title='Video Walkthrough' width='' alt='Video Walkthrough' />
+
```

Comment:
Not sure what this walkthrough is going to show to be honest.

Reply:
The "walkthrough" is a very short showing of what the app looks like when it is opened up, just to give the user an idea.  It can be removed if that is the preference.

Reply:
Since I am just starting, I'm not sure I know enough about the project to give a really meaningful walkthrough.  Perhaps this could be a placeholder until a more seasoned user creates one?  I think a quick initial setup of features would be good for new users.

Reply:
Ti ringrazio,a non sono interessato